pkg/runner/cluster_k8s.go

```diff
@@ -657,6 +656,24 @@ func (c *ClusterK8sRunner) ensureCollectOutputsPod(ctx context.Context, input *a
 	return nil
 }
 
+func (c *ClusterK8sRunner) getPodLogsStream(ctx context.Context, podName string) (io.ReadCloser, error) {
```

Comment:
Why not remove getPodLogs? at https://github.com/celestiaorg/testground/pull/1/files#diff-02c408dffd6979e1dfed096c9c3e1428017ec538ff4dc43b69f74009f451357bR677

Reply:
this is mostly a test(dirty) solution. As we discussed, I'm not the it will work as described in documentation. I will clean the code once will have a confirmation that solution is working as expected

Reply:
Yes.
Instead of getting all logs in the end of `run` function, we are spawning a routine for each instance that will get logs once they are appeared. I am a bit confused to method naming. To my understanding `Stream` is something that allows to get streaming data. But everything is working on http req/resp under the hood and http should not support streaming, iirc.